src/entities/trades/gnosis-protocol/CoWTrade.ts

```diff
@@ -26,6 +24,13 @@ import { CoWTradeError } from './CoWTradeError'
 import { CoWTradeGetBestTradeExactInParams, CoWTradeGetBestTradeExactOutParams, CoWTradeParams } from './types'
 
 const ZERO_PERCENT = new Percent(ZERO, ONE)
+enum CowChainId {
+  MAINNET = 1,
+  RINKEBY = 4,
+  GOERLI = 5,
+  GNOSIS = 100,
+  SEPOLIA = 11155111,
+}
```

Comment:
Is it possible to import this values?

Reply:
Yes! Already did it! Also added Sepolia to our ChainID constant

Reply:
Sorry, I was to vague. Import this values from cow-sdk. They must have their own ChaindId on the SDK

Reply:
As we talked offline:
We will keep the current version of the code as CoW SDK provides a ChainID enum that lacks some of the chain IDs needed for this use case.